public/chromeos.html

```diff
@@ -0,0 +1,282 @@
+                c-8.4,4.1-15.7,7.4-16.3,7.3C187,40.4,180.1,37.1,172.3,33.1z"
+                />
+              </svg>
+              <p>Xbox</p>
+            </a>
+          </li>
+          <li>
+            <a href="/playstation">
+              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 154.8">
+                <path
+                  class="svg-fill"
+                  d="M197.2,118c-3.9,4.9-13.3,8.4-13.3,8.4l-70.5,25.3V133l51.9-18.5c5.9-2.1,6.8-5.1,2-6.7 c-4.8-1.6-13.4-1.1-19.3,1L113.4,121v-19.4l2-0.7c0,0,10-3.5,24-5.1c14-1.5,31.2,0.2,44.7,5.3C199.4,106,201.1,113.1,197.2,118z M120.1,86.2V38.4c0-5.6-1-10.8-6.3-12.2c-4-1.3-6.5,2.5-6.5,8.1v119.6L75,143.6V1c13.7,2.5,33.7,8.6,44.4,12.2 c27.3,9.4,36.6,21,36.6,47.3C156,86.2,140.2,95.9,120.1,86.2z M15.9,131c-15.6-4.4-18.2-13.6-11.1-18.8c6.6-4.9,17.8-8.5,17.8-8.5 l46.2-16.4v18.7l-33.3,11.9c-5.9,2.1-6.8,5.1-2,6.7c4.8,1.6,13.4,1.1,19.3-1l16-5.8v16.8c-1,0.2-2.1,0.4-3.2,0.5 C49.6,137.6,32.6,136.5,15.9,131L15.9,131z"
+                />
+              </svg>
+              <p>PS4/5</p>
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h1 class="main-title">How to take a screenshot on ChromeOS</h1>
+      <article id="full">
+        <section class="intro">
+          <h1 class="intro-title">Screenshot Region</h1>
+          <p>Some chromebooks have a dedicated <kbd>Screenshot</kbd> key.</p>
+        </section>
+        <section class="step" id="region-1">
+          <div class="step-half step-text">
+            <h2 class="step-title">1</h2>
+            <p>
+              Press the <kbd>Screenshot</kbd> key. If your chromebook does not
+              have this key, press <kbd>Shift</kbd> + <kbd>Control</kbd> +
+              <kbd>Show Windows</kbd>. Some keyboards do not have the
+              <kbd>Show Windows</kbd> key, and you should substitute it with the
+              <kbd>F5</kbd> key.
+            </p>
+          </div>
+          <div class="step-half step-diagram">
+            <svg
```

Comment:
Just pointing out that I really appreciate you putting in the time to make graphics for this new page. I remember how much work I put into making these original images, so for you to create two new ones is amazing. ❤️